### PR TITLE
PLT-2674 Private Group should not have Leave option when only one mem…

### PR DESCRIPTION
**Pull request by enahum**
…ber remains

### CODE REVIEW

**Comment by jwilander** [comment]
+1

**Comment by crspeller** [comment]
@enahum Should this be restricted server side as well? Or do we already do this?

**Comment by enahum** [comment]
@crspeller server side validation has been added

**Comment by jwilander** [comment]
Got a gofmt failure:
```
Running GOFMT
diff api/channel.go gofmt/api/channel.go --- /tmp/gofmt726680376	2016-05-05 14:26:58.629545999 +0000 +++ /tmp/gofmt120629559	2016-05-05 14:26:58.629545999 +0000 @@ -594,7 +594,7 @@  			return  		}   -		if channel.Type == model.CHANNEL_PRIVATE  && membersCount <= 1 { +		if channel.Type == model.CHANNEL_PRIVATE && membersCount <= 1 {  			c.Err = model.NewLocAppError(leave, api.channel.leave.last_member.app_error, nil, userId=+user.Id)  			c.Err.StatusCode = http.StatusBadRequest  			return
gofmt failure
```

**Comment by enahum** [comment]
fixed.

File: api/channel_test.go [hunk 1]
```diff
@@ -485,7 +485,12 @@ func TestLeaveChannel(t *testing.T) {
 	Client.Must(Client.JoinChannel(channel1.Id))
 
 	// No error if you leave a channel you cannot see
-	Client.Must(Client.LeaveChannel(channel3.Id))
+	//Client.Must(Client.LeaveChannel(channel3.Id))
```
Comment (crspeller) [inline api/channel_test.go]: Why is this commented out?

Comment (enahum) [inline api/channel_test.go]: oops forgot to remove the comment lines. now should be ok.

**Comment by crspeller** [comment]
+1

**Pull request merged**